scout/server/blueprints/variant/templates/variant/variant_details.html

```diff
@@ -169,9 +169,13 @@
         </thead>
         <tbody>
           <tr>
+            {% if config.LOQUSDB_SETTINGS == 'Fail' %}
+              <td style="background-color: #ff9999;" colspan=3>Failed to contact loqusdb server</td>
+            {% else %}
```

Comment:
Indent?

Reply:
On the left?

Reply:
Its html, so personal preference! I would at least do the same with the two if-else blocks, like either move the above left, or move the tds below right.. 😸